Output logs docs update (#6005)
vercel/turbo: docs/pages/repo/docs/reference/command-line-reference/run.mdx

`type: string`

> Set type of output logging. Defaults to [`outputMode`](/repo/docs/reference/configuration#outputmode) for the task in `turbo.json` (if set).

Reviewer: ```suggestion
Set type of output logging. This will override [`outputMode`](/repo/docs/reference/configuration#outputmode) if defined in `turbo.json`.
```

> <OuputModeTable />

Reviewer: Can we change `OuputModeTable` here from:

```
This is the default. Displays all output
```

to this instead:

```
Displays all output (default)
```

Author: Yea good call. Done
